src/iuse_actor.cpp

```diff
@@ -2190,6 +2190,9 @@ void learn_spell_actor::info( const item &, std::vector<iteminfo> &dump ) const
     for( const std::string &sp_id_str : spells ) {
         const spell_id sp_id( sp_id_str );
         spell_text = sp_id.obj().name.translated();
+        if( sp_id->spell_class.is_valid() ) {
+            spell_text += ", " + sp_id->spell_class->name() + " ";
```

Comment:
```suggestion
            spell_text = string_format( pgettext( "spell name, spell class", "%s, %s" ), spell_text, sp_id->spell_class->name() );
```
Some languages need to localize comma and space. Also the trailing space seems unnecessary.